web/src/components/listing-details/index.tsx

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
I'm a bit confused by the folder organization. How is this "listing-details" different from the "customer/listing-details"? Why must it be in a separate folder here?

Reply:
@xlai20 Going to a particular listing page(https://gpay-group-buy.an.r.appspot.com/listing/:listingId) is a common route for both merchants and customers. The router will route the browser requesting for the listing page(https://gpay-group-buy.an.r.appspot.com/listing/:listingId) to the `components/listing-details` page.
However, we want merchants and customers to see different things when they are on https://gpay-group-buy.an.r.appspot.com/listing/:listingId. This is where we have `components/customer/listing-details` and `components/merchant/listing-details`. The `components/listing-details` page will be in charge of showing the visitor `components/customer/listing-details` or  `components/merchant/listing-details` depending on their identity (whether they are a customer or a merchant).